chromium_src/ios/chrome/browser/application_context/application_context_impl.mm

```diff
@@ -398,7 +398,8 @@ void BindNetworkChangeManagerReceiver(
 PromosManager* ApplicationContextImpl::GetPromosManager() {
   DCHECK(thread_checker_.CalledOnValidThread());
```

Comment:
do we actually need this code here? Can we just return nullptr or make a dummy PromosManager subclass that does nothing?

Reply:
cc: @kylehickinson @Brandon-T do you know if this code is ever called?

Reply:
AFAIK this isn't called anywhere by us and its usage in Chromium seem to be in Chrome iOS UI code which we don't use.

Reply:
OK great, in that case I'll modify it to return `nullptr` instead and we can verify things look good in the QA builds.

Reply:
Fixed in 7ea2b6a4468c6927ae4e8659944ac00bc600e3dd.